lib/pages/chat_blank/chat_blank.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:fluffychat/pages/chat_blank/chat_blank_style.dart';
+import 'package:fluffychat/resource/image_paths.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:go_router/go_router.dart';
+
+class ChatBlank extends StatelessWidget {
+  final bool loading;
+  const ChatBlank({this.loading = false, Key? key}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // Add invisible appbar to make status bar on Android tablets bright.
+      appBar: AppBar(
+        automaticallyImplyLeading: false,
+        elevation: 0,
+        backgroundColor: Theme.of(context).scaffoldBackgroundColor,
+      ),
+      extendBodyBehindAppBar: true,
+      body: loading ? _buildLoading(context) : _buildNotChatWidget(context),
```

Comment:
you should create private class for that
![Screenshot 2023-09-18 at 08 22 36](https://github.com/linagora/twake-on-matrix/assets/12546908/ab51b1e5-3ef2-41b5-88c8-322e9d61bb7f)
https://medium.com/@panuj330/pushing-flutter-to-the-limit-performance-tips-every-developer-should-know-87e3d835cd49